src/main/java/com/team01/carsharingapp/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package com.team01.carsharingapp.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+
+public record UserLoginRequestDto(
+        @NotBlank(message = "required field")
+        @Email(message = "must match the email format")
+        String email,
+        @NotBlank(message = "required field")
+        String password
```

Comment:
> Maybe set Size for fields? (not less … and not longer than…)

I didn't add additional restrictions because we already validate data at the user registration stage, not sure if this is necessary